subsys/bluetooth/gatt_pool.c

```diff
@@ -414,7 +379,7 @@ int bt_gatt_pool_desc_alloc(struct bt_gatt_pool *gp,
 }
 
 int bt_gatt_pool_ccc_alloc(struct bt_gatt_pool *gp,
-			   struct _bt_gatt_ccc const *ccc)
+			   struct _bt_gatt_ccc *ccc)
```

Comment:
Why this cannot be const anymore? I cannot see a reason directly in this code changes.

Reply:
Previously we made a copy of this one, and assigned attr->user_data to point to the copy, now it points to this object. It has to be modifiable now.